S7.Net/PLC.cs

```diff
@@ -44,17 +47,25 @@ public partial class Plc : IDisposable
         /// <returns>A <see cref="T:System.Int32" /> that specifies the amount of time, in milliseconds, that will elapse before a read operation fails. The default value, <see cref="F:System.Threading.Timeout.Infinite" />, specifies that the read operation does not time out.</returns>
         public int ReadTimeout
         {
-            get;
-            set;
-        } = System.Threading.Timeout.Infinite;
+            get => _readTimeout;
+            set
+            {
+                _readTimeout = value;
+                ConfigureConnection();
```

Comment:
This would set both, let's just change this to
`if (tcpClient != null) tcpClient.ReadTimeout = value;`

Reply:
Yeah, I was think about it but didn't want to duplicate this code. Maybe one line is not bad :).